Makefile

```diff
@@ -167,19 +163,20 @@ gen-test-golden: test-unit
 # dependencies to the go.mod file. To avoid that we change to a directory
 # without a go.mod file when downloading the following dependencies
 
-$(GOX):
-	(cd /; go install github.com/mitchellh/gox@v1.0.2-0.20220701044238-9f712387e2d2)
-
 $(GOIMPORTS):
 	(cd /; go install golang.org/x/tools/cmd/goimports@latest)
 
 # ------------------------------------------------------------------------------
 #  release
 
 .PHONY: build-cross
-build-cross: LDFLAGS += -extldflags "-static"
-build-cross: $(GOX)
-	GOFLAGS="-trimpath" CGO_ENABLED=0 $(GOX) -parallel=3 -output="_dist/{{.OS}}-{{.Arch}}/$(BINNAME)" -osarch='$(TARGETS)' $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' ./cmd/helm
+build-cross:
+	@echo "==> Building cross-platform binaries using GoReleaser"
+	goreleaser build --clean --skip-validate --single-target
```

Comment:
`--skip-validate` is deprecated -- see: https://goreleaser.com/deprecations/

don't you want/need to refer to the config file? And why is `--single-target` being passed here?